examples/spark/README.md

```diff
@@ -119,96 +157,104 @@ program.
+pods.) You can also interrogate the status in the following way:
+
+```console
 $ kubectl get pods
-NAME                                            READY     STATUS    RESTARTS   AGE
-[...]
-spark-master                                    1/1       Running   0          14m
-spark-worker-controller-hifwi                   1/1       Running   0          33s
-spark-worker-controller-u40r2                   1/1       Running   0          33s
-spark-worker-controller-vpgyg                   1/1       Running   0          33s
+NAME                            READY     STATUS    RESTARTS   AGE
+spark-master-controller-5u0q5   1/1       Running   0          25m
+spark-worker-controller-e8otp   1/1       Running   0          6m
+spark-worker-controller-fiivl   1/1       Running   0          6m
+spark-worker-controller-ytc7o   1/1       Running   0          6m
 
-$ kubectl logs spark-master
+$ kubectl logs spark-master-controller-5u0q5
 [...]
-15/06/26 14:15:43 INFO Master: Registering worker 10.244.2.35:46199 with 1 cores, 2.6 GB RAM
-15/06/26 14:15:55 INFO Master: Registering worker 10.244.1.15:44839 with 1 cores, 2.6 GB RAM
-15/06/26 14:15:55 INFO Master: Registering worker 10.244.0.19:60970 with 1 cores, 2.6 GB RAM
+15/10/26 18:20:14 INFO Master: Registering worker 10.244.1.13:53567 with 2 cores, 6.3 GB RAM
+15/10/26 18:20:14 INFO Master: Registering worker 10.244.2.7:46195 with 2 cores, 6.3 GB RAM
+15/10/26 18:20:14 INFO Master: Registering worker 10.244.3.8:39926 with 2 cores, 6.3 GB RAM
 ```
 
 ## Step Three: Start your Spark driver to launch jobs on your Spark cluster
 
 The Spark driver is used to launch jobs into Spark cluster. You can read more about it in
-[Spark architecture](http://spark.apache.org/docs/latest/cluster-overview.html).
+[Spark architecture](https://spark.apache.org/docs/latest/cluster-overview.html).
 
-```shell
-$ kubectl create -f examples/spark/spark-driver.json
+```console
+$ kubectl create -f examples/spark/spark-driver-controller.yaml
+replicationcontrollers/spark-driver-controller
 ```
 
 The Spark driver needs the Master service to be running.
 
 ### Check to see if the driver is running
 
-```shell
-$ kubectl get pods
-NAME                                           READY     REASON    RESTARTS   AGE
-[...]
-spark-master                                    1/1       Running   0          14m
-spark-driver                                    1/1       Running   0          10m
+```console
+$ kubectl get pods -lcomponent=spark-driver
+NAME                            READY     STATUS    RESTARTS   AGE
+spark-driver-controller-vwb9c   1/1       Running   0          1m
 ```
 
 ## Step Four: Do something with the cluster
 
-Use the kubectl exec to connect to Spark driver
+Use the kubectl exec to connect to Spark driver and run a pipeline.
 
-```
-$ kubectl exec spark-driver -it bash
-root@spark-driver:/#
-root@spark-driver:/# pyspark
+```console
+$ kubectl exec spark-driver-controller-vwb9c -it pyspark
```

Comment:
My next goal is actually to get Zeppelin up (because in terms of visual demos, it works much, much better), so don't stress too much about this one. I *believe* the driver pod also has spark-shell on it, but I need to poke at it.